scheduler/config-k8s.edn

```diff
@@ -35,16 +35,19 @@
  :hostname #config/env "COOK_HOSTNAME"
  :kubernetes {:disallowed-container-paths #{"/mnt/bad"}
               :disallowed-var-names #{"BADVAR"}
-              :init-container {:command ["/bin/sh" "-c" "echo sample init container running"]
-                               :image "byrnedo/alpine-curl:latest"}
-              :sandbox-fileserver {:command ["fileserver"]
-                                   :image "twosigma/cook-fileserver:latest"
-                                   :port 23906
-                                   :resource-requirements {:cpu-request 0.1
-                                                           :cpu-limit 0.1
-                                                           :memory-request 128.0
-                                                           :memory-limit 128.0}}
-              :custom-shell ["/bin/sh" "-c"]}
+              ;; The job resource adjustment causes config/default-pool to return nil. Until
+              ;; we fix that, we should run without the sidecar.
+              ;:init-container {:command ["/bin/sh" "-c" "echo sample init container running"]
+              ;                 :image "byrnedo/alpine-curl:latest"}
+              ;:sandbox-fileserver {:command ["fileserver"]
+              ;                     :image "twosigma/cook-fileserver:latest"
+              ;                     :port 23906
+              ;                     :resource-requirements {:cpu-request 0.1
+              ;                                             :cpu-limit 0.1
+              ;                                             :memory-request 128.0
+              ;                                             :memory-limit 128.0}}
+              ;:custom-shell ["/bin/sh" "-c"]
+              }
```

Comment:
We should put the comment in a TODO if we choose to leave the code commented out.

Else I would vote to remove the code altogether.
Same for the commented code below.

Reply:
Added `TODO`s, thanks